### ISSUE TITLE
Warning when opening merged DL1 file

### ISSUE THREAD

**Issue by maxnoe** [cta-observatory/ctapipe#1611]
```
>>> from ctapipe.io import EventSource
>>> s = EventSource(path)

WARNING: Table /configuration/simulation/run is missing column event_id that is in container EventIndexContainer. It will be skipped.
```

**Comment by LukasNickel** [comment]
Actually there is. It is used to match mc headers in case of merged files.
We expect that warning to be thrown (in the sense, that there is no bug here), but we might want to silence it

**Comment by maxnoe** [comment]
Why event index? Because it can read `obs_id`?

**Comment by maxnoe** [comment]
Fixed in #1614
